src/CMakeLists.txt

```diff
@@ -80,4 +84,4 @@ if (BUILD_TEST)
 endif()
 
 # pybind bindings
-add_subdirectory(esp/bindings)
+#add_subdirectory(esp/bindings)
```

Comment:
I had some conflict with py bindings before when developing. Need to enable this back